charts/fluent-operator/templates/fluentbit-clusterinput-tail.yaml

```diff
@@ -10,7 +10,7 @@ metadata:
     fluentbit.fluent.io/component: logging
 spec:
   tail:
-    tag: kube.*
+    tag: {{ .Values.fluentbit.input.tail.tag  }}
```

Comment:
It's not enough to modify tag for tail input only.
For example, it's used by other plugins like filter and output:
https://github.com/fluent/fluent-operator/blob/master/charts/fluent-operator/templates/fluentbit-clusterfilter-kubernetes.yaml#L12

Reply:
got it. 
I will update this PR with the wider plugin lists.

Reply:
and also sign the commits.